### ISSUE TITLE
[Bug] Failed to set up - Seems to be a VPN issue

### ISSUE THREAD

**Issue by ShawnCBerg** [Vaskivskyi/ha-asusrouter#726]
### The problem

For the last few weeks through a few upgrades of ha-asusrouter the integration has reporterd "Failed to set up" on the Integrations page. I checked the logs today and saw the following:

```
Logger: homeassistant.config_entries
Source: config_entries.py:402
First occurred: 1:20:21 PM (4 occurrences)
Last logged: 1:30:14 PM

Error setting up entry 192.168.50.1 for asusrouter
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 402, in async_setup
    result = await component.async_setup_entry(hass, self)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/asusrouter/__init__.py", line 26, in async_setup_entry
    await router.setup()
  File "/config/custom_components/asusrouter/router.py", line 352, in setup
    await self._init_sensor_coordinators()
  File "/config/custom_components/asusrouter/router.py", line 616, in _init_sensor_coordinators
    available_sensors = await self.bridge.async_get_available_sensors()
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/asusrouter/bridge.py", line 208, in async_get_available_sensors
    SENSORS: await self._get_sensors_modern(AsusData.OPENVPN_CLIENT),
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/asusrouter/bridge.py", line 507, in _get_sensors_modern
    data = await self.api.async_get_data(datatype)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/asusrouter/asusrouter.py", line 566, in async_get_data
    processed = process(
                ^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/asusrouter/modules/endpoint/__init__.py", line 112, in process
    return submodule.process(data)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/asusrouter/modules/endpoint/hook.py", line 138, in process
    vpnc, vpnc_clientlist = process_vpnc(data)
                            ^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/asusrouter/modules/endpoint/hook.py", line 472, in process_vpnc
    vpn[sorted_type][sorted_id] = info
    ~~~^^^^^^^^^^^^^
KeyError: <AsusVPNType.UNKNOWN: 'Unknown'>

```

It appears there's an issue with a VPN config on my router. I did recently add a Surfshark VPN connection to my router in VPN Fusion config. This seems to be a new VPN type as I hadn't recalled seeing it before. I'm running firmware 3.0.0.4.388_23013-g3e1c174 on a ZenWifi Pro ET12. I've included a screenshot of the VPN config from the router below. 

After deleting the VPN configuration in VPN Fusion the integration loads without issue.

![CleanShot 2023-11-24 at 15 01 07@2x](https://github.com/Vaskivskyi/ha-asusrouter/assets/4161988/d20227c0-a979-4172-90cc-33478cce83db)

### Your device model

ZenWiFi Pro ET12

### Firmware type

Stock

### Firmware version

3.0.0.4.388_23013-g3e1c174

### Integration version with the issue

0.24.2

### Method of the integration installation

HACS

### What version and type of Home Assistant installation do you use

Core 2023.11.3

### Is there anything useful in the logs?

```txt
Logger: homeassistant.config_entries
Source: config_entries.py:402
First occurred: 1:20:21 PM (5 occurrences)
Last logged: 2:59:35 PM

Error setting up entry 192.168.50.1 for asusrouter
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 402, in async_setup
    result = await component.async_setup_entry(hass, self)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/asusrouter/__init__.py", line 26, in async_setup_entry
    await router.setup()
  File "/config/custom_components/asusrouter/router.py", line 352, in setup
    await self._init_sensor_coordinators()
  File "/config/custom_components/asusrouter/router.py", line 616, in _init_sensor_coordinators
    available_sensors = await self.bridge.async_get_available_sensors()
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/asusrouter/bridge.py", line 208, in async_get_available_sensors
    SENSORS: await self._get_sensors_modern(AsusData.OPENVPN_CLIENT),
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/asusrouter/bridge.py", line 507, in _get_sensors_modern
    data = await self.api.async_get_data(datatype)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/asusrouter/asusrouter.py", line 566, in async_get_data
    processed = process(
                ^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/asusrouter/modules/endpoint/__init__.py", line 112, in process
    return submodule.process(data)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/asusrouter/modules/endpoint/hook.py", line 138, in process
    vpnc, vpnc_clientlist = process_vpnc(data)
                            ^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/asusrouter/modules/endpoint/hook.py", line 472, in process_vpnc
    vpn[sorted_type][sorted_id] = info
    ~~~^^^^^^^^^^^^^
KeyError: <AsusVPNType.UNKNOWN: 'Unknown'>
```

**Comment by ShawnCBerg** [comment]
No apologies at all! Happy to contribute and report the bug and look forward to the resolution.

**Comment by ShawnCBerg** [comment]
Thanks!

**Comment by Vaskivskyi** [comment]
Released in `0.25.0`
